daemon/volumes_unix.go

```diff
@@ -64,6 +65,7 @@ func parseBindMount(spec, volumeDriver string) (*mountPoint, error) {
 		RW: true,
 	}
 	arr := strings.Split(spec, ":")
+	reg := regexp.MustCompile(`^[a-zA-Z0-9][a-zA-Z0-9_.-]+$`)
```

Comment:
I think it would be better if this were declared immediately above where it is used